src/providers/object-provider.js

```diff
@@ -210,12 +209,11 @@ export default class YamcsObjectProvider {
         return this.url + 'api/mdb/' + this.instance + '/' + operation + name;
     }
 
-    async #fetchMdbApi(operation, abortSignal) {
+    async #fetchMdbApi(operation, property, abortSignal) {
         const mdbURL = `${this.url}api/mdb/${this.instance}/${operation}`;
-        const response = await fetch(mdbURL, { signal: abortSignal });
-        const parsedJSON = await response.json();
+        const response = await accumulateResults(mdbURL, { signal: abortSignal }, property, []);
```

Comment:
Switched from a `fetch` to this `accumulateResults` function I found in utils that seems to already handle pagination